<error errorId="abdd77a1-5318-41b7-b028-1415115425dd" application="Hangfire" host="LOLA" type="Error" message="Execution BackgroundServerProcess is in the Failed state now due to an exception, execution will be retried no more than in 00:00:15" detail="System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---&gt; System.ComponentModel.Win32Exception (0x80004005): The user name or password is incorrect&#xD;&#xA;   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)&#xD;&#xA;   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)&#xD;&#xA;   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)&#xD;&#xA;   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)&#xD;&#xA;   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)&#xD;&#xA;   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)&#xD;&#xA;   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)&#xD;&#xA;   at System.Data.SqlClient.SqlConnection.Open()&#xD;&#xA;   at Hangfire.SqlServer.SqlServerStorage.CreateAndOpenConnection()&#xD;&#xA;   at Hangfire.SqlServer.SqlServerStorage.UseConnection[T](DbConnection dedicatedConnection, Func`2 func)&#xD;&#xA;   at Hangfire.SqlServer.SqlServerStorage.UseConnection(DbConnection dedicatedConnection, Action`1 action)&#xD;&#xA;   at Hangfire.SqlServer.SqlServerConnection.AnnounceServer(String serverId, ServerContext context)&#xD;&#xA;   at Hangfire.Server.BackgroundServerProcess.CreateServer(BackgroundServerContext context)&#xD;&#xA;   at Hangfire.Server.BackgroundServerProcess.Execute(Guid executionId, BackgroundExecution execution, CancellationToken stoppingToken, CancellationToken stoppedToken, CancellationToken shutdownToken)&#xD;&#xA;   at Hangfire.Server.BackgroundProcessingServer.RunServer(Guid executionId, Object state)&#xD;&#xA;   at Hangfire.Processing.BackgroundExecution.Run(Action`2 callback, Object state)&#xD;&#xA;ClientConnectionId:1944138d-405f-4d77-a961-a7a5a845931a&#xD;&#xA;Error Number:1326,State:0,Class:20" time="2019-06-24T21:21:18.6483209Z" />